plugins/terminal-proxy/lib/terminalProxy.js

```diff
@@ -259,11 +259,11 @@ TerminalWebsocketProxy.prototype.handleTerminalClientMessage = function(message,
     jsonObject = JSON.parse(message);
   } catch (e) {
     //not json
-    this.logger.warn(this.identifierString()+' sent messsage which was not JSON');
+    this.logger.warn("ZWED0133W", this.identifierString()); //this.logger.warn(this.identifierString()+' sent messsage which was not JSON');
     this.closeConnection(websocket, WEBSOCKET_REASON_TERMPROXY_INTERNAL_ERROR, 'Message not JSON');
   }
-  this.logger.debug(this.identifierString()+' Websocket client message received. Length='+message.length);
-  this.logger.log(this.logger.FINER,this.identifierString()+' Websocket client message content='+message);
+  this.logger.debug("ZWED0269I", this.identifierString(), message.length); //this.logger.debug(this.identifierString()+' Websocket client message received. Length='+message.length);
+  this.logger.debug("ZWED0135I", this.identifierString(), message); //this.logger.log(this.logger.FINER,this.identifierString()+' Websocket client message content='+message);
```

Comment:
You may not want this as debug.
The reason why it was FINER was that printing the length is a lot less intense than printing the entire contents, because this can be thousands of characters.
I believe the next level is this.logger.trace, whereas this.logger.debug is the brief version.

Reply:
Changed to trace for the data.